cmd/utils.go

```diff
@@ -342,83 +341,46 @@ func checkVersion(config map[string]interface{}) error {
 	return nil
 }
 
-// Parse parses flag settings from YAML config file and command line.
-func Parse(configPath, serviceName string) error {
-	err := ParseFlagsWithConfig(flag_.CommandLine, os.Args[1:], configPath, serviceName)
-	if err == ErrDumpRequested {
-		DumpConfig(configPath, serviceName, true)
-		os.Exit(0)
+// ParseFlags parses CommandLine flags and dump config if requested.
```

Comment:
this function just parse flags or return error otherwise, and return ErrDumpRequested on dump config arg. It doesn't do dumping